ng-dev/utils/config.ts

```diff
@@ -114,7 +115,7 @@ function readConfigFile(configPath: string, returnEmptyObjectOnError = false): o
     // dev-infra tool runs in NodeJS which does not support ES modules by default.
     // Additionally, set the `dir` option to the directory that contains the configuration
     // file. This allows for custom compiler options (such as `--strict`).
-    require('ts-node').register({
+    register({
```

Comment:
We intentionally load this dynamically as otherwise we'd always require `ts-node`. I'm not sure where we stand here. I guess all configs use TypeScript but seems like this would be better in a separate change cleaning this up completely if we go that way.